data/brands/shop/supermarket.json

```diff
@@ -1855,6 +1855,17 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Economy Cash",
+      "locationSet": {"include":["es"]},
+      "tags":{
+        "brand": "Economy Cash",
+        "operator": "Kuups Design International SL",
```

Comment:
The same problem as https://github.com/osmlab/name-suggestion-index/pull/7274/files#r1008737352

(You don't need to remove operator/operator:wikidata)

Below "Vidal Supermercados" also missing a brand:wikidata